### DISCUSSION TITLE
recipes/desserts/corn-%26-hatch-chile-cupcakes-with-honey-buttercream/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#338] [Comments]
# recipes/desserts/corn-%26-hatch-chile-cupcakes-with-honey-buttercream/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/corn-%26-hatch-chile-cupcakes-with-honey-buttercream/

**Comment by nicholaswilde** [comment]
Separate instructions
